tests/validate_docs.sh

```diff
@@ -41,6 +41,11 @@ for term in $(sed -n 's/[ ]\+{"\([a-z0-9-]\+\)", YAML_[A-Z]\+_NODE.*/\1/p' src/p
         continue
     fi
 
+    # 6. others is an undocumented fallback/passthrough feature
```

Comment:
Just an idea: maybe we should actually document this, but mention in the documentation that it's only for internal usage? Not sure what our approach to this is generally, but if it's undocumented then people might be all like 'WTF' when reading the NM-generated yaml neplan code.